lib/listing.rb

```diff
@@ -0,0 +1,42 @@
+class Listing
```

Comment:
`Listing` sounds equivalent to `row` to me...how about `Table` or `ListingSet` or something?